### ISSUE TITLE
Sonarr detects 1080p episode as 720p

### ISSUE THREAD

**Issue by PeterPan669** [Sonarr/Sonarr#3522]
Sonarr dl'ed a file, called <series>.S01E15.<Episode.Name>.BluRay.1080p.DD5.1.H265-d3g.1.
It then 'determines' that the file is 720p. 

**Screenshots**
![image](https://user-images.githubusercontent.com/59241040/72478233-c893d900-37f1-11ea-9653-dc12cf7abaf8.png)

![image](https://user-images.githubusercontent.com/59241040/72478274-dba6a900-37f1-11ea-8643-c2b6200b87e4.png)

**System Information**
 - Sonarr Version: 3.0.3.688
 - Operating System: Synology DSM6
 - .net Framework (Windows) or mono (macOS/Linux) Version: 5.18.0.240

**Additional context**
The file really is Bluray-1080p. Shouldn't it mark it as such?
Sonarr did the dl, renaming, etc. No manual actions on this. Yet due to this bug, it keeps looking for the quality I specified while it already has it.

**Comment by markus101** [comment]
Going to need to see debug logs of the import.

Based on the screen shots the source is also wrong, which usually happens when the quality is missing from the file name, but media info is used to get resolution, which means the resolution of the file would also be wrong.

**Comment by PeterPan669** [comment]
Hi Markus,
What u mean "the source is also wrong"? The quality is in the filename (...BluRay.1080p.DD5.1.H265-d3g.1.mkv). The resolution is is 1920 x 1080.
![image](https://user-images.githubusercontent.com/59241040/72482663-cd12be80-37fe-11ea-9fc8-64b5e086dcc7.png)

This is happening a lot btw. I'll see if I can dig up some debug logs.

**Comment by markus101** [comment]
Source: HDTV vs Bluray

In your history screenshot I see the grab, but not the import are those screenshots from the same episode? If so, how'd the file end up in the series folder, because it doesn't look like Sonar imported it.

**Comment by PeterPan669** [comment]
Here's the debug log.
[debug.txt](https://github.com/Sonarr/Sonarr/files/4068600/debug.txt)

I already had it downloaded a couple of days ago. When I saw the 720p quality label, I figured something went wrong and searched for it again. However, turns out it did download the 1080p version, and it won't import since it's the same file / size.

**Comment by markus101** [comment]
> I already had it downloaded a couple of days ago. 

Does your screenshot just exclude that then? Or was it downloaded a long time ago and the existing file name lacks the quality?

> However, turns out it did download the 1080p version, and it won't import since it's the same file / size.

Your logs show that it's parsing the quality fine, `20-1-16 01:30:49.1|Debug|Parser|Quality parsed: Bluray-1080p v1` the fact that the old file was detected wrong is impossible to diagnose without debug logs, but the only way Sonarr would lose track of the quality would be importing the file again (because it was deleted/went missing) and `Analyse video files` is disabled.

Rejecting because it's the same filesize is expected, though the plan is to eventually remove that in favour of some smarter tracking of quality from grab not matching the quality of the file at import time.

**Comment by PeterPan669** [comment]
I see what you mean, I don't have any import action in the History (compared to a recent episode). I also had the Analyse video files disabled from the getgo. Most likely this is due to something funky on my side, not sure how it happened. 
For my understanding:
- I've enabled "Analyse video files" now; Would refresh & scan / Update all get the correct quality status from the existing files? I notice that at this moment, the media info gives the correct quality on the tooltip, but the label still says "HDTV-720":

![image](https://user-images.githubusercontent.com/59241040/72484309-13b6e780-3804-11ea-81ae-3c5397348a94.png)

![image](https://user-images.githubusercontent.com/59241040/72484315-1a455f00-3804-11ea-8f44-967d2cf79561.png)

2. Would this 'wrong' label prevent me in any way to get / update files to the wanted quality profile that I configured?

Thanks again.
